compiler/ztests/sql/cte-recursive.yaml

```diff
@@ -0,0 +1,16 @@
+spq: |
+  WITH t AS (
+    SELECT 1
+      UNION ALL
+    SELECT x+1
+    FROM   t
+    WHERE  x < 200000
+  )
+  SELECT count(t1) FROM t AS t1;
+
+input: null
+
+error: |
+  recursive WITH relations not currently supported at line 5, column 10:
```

Comment:
The real error here is that the CTE refers to itself but doesn't include the RECURSIVE keyword.

Reply:
Yeah but we also don't support the RECURSIVE keyword so I wanted to avoid telling them to use the RECURSIVE keyword then have it not work.